src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -821,29 +739,31 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
                 conNew.Phone != conOld.Phone ||
                 conNew.Fax != conOld.Fax);
     }
- 
+
     /*******************************************************************************************************
     * @description returns whether changes in the contact require a rename of a HH Account
     * @param hhNamingFields the set of fields used in the namespec
     * @param conNew the new version of the Contact
     * @param conOld the old version of the Contact
-    * @return boolean
+    * @return Boolean
     */
-    private boolean needsHHAccountRename(Set<String> hhNamingFields, Contact conNew, Contact conOld) {
-        if (conNew == null && conOld != null)
-            return true;
-        if (conNew != null && conOld == null)
+    private Boolean needsHHAccountRename(Contact conNew, Contact conOld) {
+        if (conNew.npe01__Organization_Type__c != CAO_Constants.HH_ACCOUNT_TYPE) {
+            return false;
+        }
+        if (conNew != null && conOld == null) {
             return true;
+        }
         if (conNew.FirstName == null) conNew.FirstName = '';
         if (conNew.LastName == null) conNew.LastName = '';
-        
+
```

Comment:
Should the two above `if` block be split out into multiple lines with curly brackets?

Reply:
I did not fix all of the other methods, just split out the method I was working in. I can go through and do some cleanup on the other methods too, like names and capitalization.